### DISCUSSION TITLE
macOS Sequoia Public Beta: KeyClu Settings Opens Up Every Time I Start the KeyClu Application

### DISCUSSION THREAD

**Discussion by ghost** [Anze/KeyCluCask#86] [Q&A]
Every time I start the KeyClu application (including on computer startup) in macOS Sequoia Public Beta, this settings window opens. Will there be a fix to this problem, especially for the macOS Sequoia Public Beta?
![Screen Shot 2024-07-19 at 11 58 51 PM](https://github.com/user-attachments/assets/5803e472-fb1a-41c5-acf5-3024302197fb)

**Comment by Anze** [comment]
Hey [awseefrahman](https://github.com/tawseefrahman),
Thank you for reporting the issue. I will investigate the compatibility with the latest macOS version.
